src/pages/staking/index.js

```diff
@@ -1,422 +1,758 @@
+                  <li>
+                    In general, risks consist of a combination of counter-party,
+                    smart contract and execution risk
+                  </li>
+                </ul>
+              </div>
+              <div style={{ gridArea: "pool-reqs" }}>
+                <h4>Requirements</h4>
+                <ul>
+                  <li>
+                    Lowest ETH requirements, some projects require as little as
+                    0.01 ETH
                   </li>
                   <li>
-                    <Translation id="page-staking-upgrades-li-5" />
+                    Deposit directly from your wallet to different pooled
+                    staking platforms or simply trade for one of the staking
+                    liquidity tokens
                   </li>
                 </ul>
-              </BoxText>
-            </Box>
-          </Column>
-        </Row>
-        <H2>
-          <Translation id="page-staking-benefits" />
-        </H2>
-        <CardContainer>
-          <StyledCard
-            emoji=":evergreen_tree:"
-            title={translateMessageId("page-staking-sustainability", intl)}
-            description={translateMessageId(
-              "page-staking-sustainability-desc",
-              intl
-            )}
-          />
-          <StyledCard
-            emoji=":globe_showing_americas:"
-            title={translateMessageId("page-staking-accessibility", intl)}
-            description={translateMessageId(
-              "page-staking-accessibility-desc",
-              intl
-            )}
-          />
-          <StyledCard
-            emoji=":old_key:"
-            title={translateMessageId("page-staking-sharding", intl)}
-            description={translateMessageId("page-staking-sharding-desc", intl)}
-          >
-            <Link to="/upgrades/shard-chains/">
-              <Translation id="page-staking-more-sharding" />
-            </Link>
-          </StyledCard>
-        </CardContainer>
-      </Content>
-    </Page>
+              </div>
+              <div style={{ gridArea: "pool-cta" }}>
+                <StyledButtonLink
+                  onClick={() => {
+                    trackCustomEvent({
+                      eventCategory: `Staking Comparison`,
+                      eventAction: `Clicked`,
+                      eventName: "clicked pooled staking",
+                    })
+                  }}
+                  to="/staking/pools/"
+                >
+                  More on pooled staking
+                </StyledButtonLink>
+              </div>
+            </ComparisonGrid>
+          </Content>
+          <Divider />
+          <StakingCommunityCallout id={tocItems.joinTheCommunity.id} />
+          <Content>
+            <h2 id={tocItems.faq.id}>{tocItems.faq.title}</h2>
+            <ExpandableCard title="What is a validator?">
+              A <em>validator</em> is a virtual entity that lives on the Beacon
```

Comment:
This answer feels a bit too in the weeds IMO, e.g.  I don't think we should be explaining effective balance in the definition of a validator.

I'd say let's use something simpler like in https://launchpad.ethereum.org/en/faq#introduction

> A validator is an entity that participates in the consensus of the Ethereum protocol.

Reply:
I honestly really don't think I would consider this "too in the weeds" for solo staking. These are basic concepts that need to be understood, since users often think they can deposit more than 32 ETH to a validator to increase their rewards. 

Think I'd suggest breaking this into a separate FAQ question like "Can I deposit more than 32 ETH?"... and providing the note and link about "effective balance" there.

Reply:
Agreed for the solo taking page but this in on the staking homepage.

Reply:
I think a dedicated FAQ to "Can I deposit more than 32 ETH?" makes sense 👍